test/externalTests/brink.sh

```diff
@@ -37,8 +37,8 @@ function test_fn { SNAPSHOT_UPDATE=1 npx --no hardhat test; }
 function brink_test
 {
     local repo="https://github.com/brinktrade/brink-core"
-    local ref_type=branch
-    local ref=master
+    local ref_type=release
+    local ref=""
```

Comment:
The fact that `ref` is empty looks a bit confusing. Maybe it would be better to change `ref_type` to `latest_release`? Or require `ref=latest` or something like that?

Reply:
Yeah, it could be `ref` latest I guess. The idea, is that it would actually never be empty, thus that assertion. Or it was provided in the script or retrieve by the `git tag` command, otherwise something wrong happened.

Reply:
> The idea is that it would actually never be empty, thus that assertion in the `common.sh` that you mentioned here

Yeah, but you can still assert that it's `latest` and then overwrite it with the actual latest tag. You might also use something like `<latest>` to make it more apparent that it's just a placeholder.

Reply:
Ok, I will do that. But I guess the assert need to be after the `ref` update anyway, because if the `git tag` for some reason returns empty, the `git checkout` will be a no-op, and it should fail instead.

Reply:
Well, you need both actually. Before update `ref` should be `<latest>` when `ref_type` is `release`. After the update you assert that it's non-empty.

I mean, unless you remove `ref_type`. Then you can't check that first condition any more. But then it does not matter because `<latest_release>` or something like that actually indicates that it's a release and no need to check anything else.